docs/B-Computations/expressions.md

```diff
@@ -22,7 +26,7 @@ The ALU evaluates the simplest of instructions on integer values: for instance,
 
 The ALU receives the expression's operator from the Control Unit, applies that operator to integer values stored in the CPU's registers and places the result in one of the CPU's registers. The FPA does the same but for floating-point values.
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/alu.png)
+![Diagram of relationship between ALU and FPA](https://ict.senecacollege.ca//~ipc144/pages/images/alu.png)
```

Comment:
Since this image is relatively small, please look into centering it horizontally:
![image](https://user-images.githubusercontent.com/52294825/142274981-c2d1217d-fa8b-4c76-9c49-5fe705639549.png)